f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -176,8 +176,10 @@ def add_ssl_profile(self, tls, vip, bigip):
     def _create_ssl_profile(
             self, container_ref, bigip, vip, sni_default=False):
         cert = self.cert_manager.get_certificate(container_ref)
-        key = self.cert_manager.get_private_key(container_ref)
         intermediates = self.cert_manager.get_intermediates(container_ref)
+        key = self.cert_manager.get_private_key(container_ref)
+        key_passphrase = \
```

Comment:
If line width > 79, I think renaming your var to 'pass' should be fine.

Reply:
`paas` is a Python keywords